app/controllers/api/v1/base_controller.rb

```diff
@@ -1,10 +1,12 @@
-class Api::V1::BaseController < ApplicationController
+class Api::V1::BaseController < ActionController::API
```

Comment:
We can have same parent `ApplicationController`. API only apps have `ActionController::API` as their parent class. But as we are already inside normal Rails app, we can use `ApplicationController`.